components/ContactFAQ.tsx

```
@@ -0,0 +1,169 @@
+    itemId: string,
+    index: number
+  ) => {
+    switch (e.key) {
+      case "Enter":
+      case " ":
+        e.preventDefault();
+        toggleItem(itemId);
+        break;
+      case "ArrowDown":
+        e.preventDefault();
+        const nextIndex = index + 1;
```

Biome (2.4.15)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)